### ISSUE TITLE
Help with reproducing DSTC8 results

### ISSUE THREAD

**Issue by alexcoca** [ypruan/DSTC8-SGDST#1]
Thank you for making your code publicly available. I was wondering if you could update the instructions on how to run your code in order to reproduce the results published in your very interesting paper [here](https://arxiv.org/pdf/2002.00181.pdf). I would appreciate if you could update the readme to reflect the sequence of commands that will reproduce the results in Table 8.

My concern is that, for example, most of the training scripts appear to import the `Corpus` object from `data_utils_test` as opposed to `data_utils_v2` and this object. If the path to the `dataset` folder is sent to this object, then the test set will be loaded twice and training will be done on the test set. Am I expected to change the import statement (or alternatively the paths inside the `Corpus.__init__`)?

Thank you for your very interesting work, I hope you will be able to support the wider research community with developing your insightful work.

**Comment by Bovey0809** [comment]
Did you get the weights? I can't get the results as the paper , it's only 54% acc for the dev set.
